lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -1414,7 +1418,7 @@ ThreadContext::SetForceOneIdleCollection()
 BOOLEAN
 ThreadContext::IsOnStack(void const *ptr)
 {
-
+#ifdef _WIN32
```

Comment:
Discussed offline, @digitalinfinity meant __readfsdwords/etc. are VC intrinsic functions and should be under _MSC_VER. Will add.